nautobot/dcim/migrations/0041_fixup_null_statuses.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+from nautobot.extras.utils import fixup_null_statuses
+
+
+def migrate_null_statuses(apps, schema):
+    status_model = apps.get_model("extras", "Status")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    for model_name in (
```

Comment:
Wonder if we can just do a `FeatureQuery("status").get_query()` or something similar here instead of hardcoding the names into the for loop?

Reply:
Could do that if we're okay with migrating models from multiple apps in a single migration, which I *think* we would want to avoid? Otherwise I'd have to take the FeatureQuery and filter it down again by app_label which seems like an unnecessary layer of indirection.